projects/kit/components/input-tag/input-tag.component.ts

```diff
@@ -210,7 +211,9 @@ export class TuiInputTagComponent
 
     @HostBinding('class._label-outside')
     get labelOutside(): boolean {
-        return this.controller.labelOutside;
+        const {size, labelOutside} = this.controller;
+
+        return sizeBigger(size, 's') ? labelOutside : true;
```

Comment:
I think we can just use `size === 's'` here

Reply:
done